### ISSUE TITLE
BlazePose

### ISSUE THREAD

**Issue by Crusoekid** [PINTO0309/PINTO_model_zoo#27]
why not translate 'pose_landmark_upper_body.tflite' to pb,'pose_landmark_upper_body_256x256_float32' is not the standard model in google project.Can you translate'pose_landmark_upper_body.tflite' to pb? thanks a lot

**Comment by PINTO0309** [comment]
”pose_landmark_upper_body_256x256_float32” completely reproduces the standard model provided by Google. I don't know what you are expecting.

**Comment by Crusoekid** [comment]
ok, when i use the model ,the output name is very strange, also, my points result can`t show at corret position.Maybe is my fault.

**Comment by PINTO0309** [comment]
The output name is read as Identity during the reverse conversion of tflite, but there is no difference in the output value.

**Comment by Crusoekid** [comment]
the output name is 'model/tf.reshape/Reshape' ,very strange!

**Comment by vpenades** [comment]
@Crusoekid Usually tflite and PB produce slightly different names, so in general, don't expect PB and TFLite to have exactly the same names.

**Comment by alex89607** [comment]
I'm trying to start training the BlazePose Full Keypoints model from scratch, but it doesn't work well. Perhaps there are some Is training data or schema for use, so that I can prepare my own?

**Comment by vidheyoza** [comment]
Also the outputs from 058 (BlazePose Full) and 053 (BlazePose) have outputs as classifiers and regressors but no information is given on how to convert them to the 33x3 key point array as given in the paper. Can anyone guide me on that?

**Comment by PINTO0309** [comment]
The following repositories are helpful.
**https://github.com/terryky/tfjs_webgl_app/tree/master/blazepose_fullbody**

**Comment by alankar66** [comment]
Hello there,
Thanks for the amazing repo.

I am working on Blazepose_full_Keypoints model and when trying to predict output from the "full_pose_landmark_39kp.h5"  model weight file and while passing (X,Y) predicted coordinates on the image, it only displays 5 keypoints and they were also wrong.

Could you please help me , how to visualize them correctly.

**Comment by vladmandic** [comment]
fyi, i've implemented latest blazepose full (155 values as 31 points with 5 properies) from <https://github.com/PINTO0309/PINTO_model_zoo/tree/main/058_BlazePose_Full_Keypoints/10_new_256x256>
and blazepose upperbody (155 values as 31 points with 5 properies) from <https://github.com/PINTO0309/PINTO_model_zoo/tree/main/053_BlazePose/20_new_256x256>  
btw, `visibility` and `presence` properies are interpreted as reverse sigmoid value.

js code is at <https://github.com/vladmandic/human/blob/main/src/blazepose/blazepose.ts>  
and with some try&error, i've created annotations for both variations: <https://github.com/vladmandic/human/blob/main/src/blazepose/annotations.ts>
